doc/rados/deployment/ceph-deploy-mds.rst

```diff
@@ -2,10 +2,6 @@
  Add/Remove Metadata Server
 ============================
 
-With ``ceph-deploy``, adding and removing metadata servers is a simple task. You
-just add or remove one or more metadata servers on the command line with one
-command.
-
```

Comment:
These documents in `/doc/rados/deployment/` are all relating to ceph-deploy. I don't think you should remove this. In fact, I'm not even sure if updating this documents are worth the trouble since ceph-deploy is legacy and unmaintained.

@joscollin why not just create a new document in `doc/cephfs/` which details what it means to add/remove an MDS.